src/main/java/frc/team2412/robot/commands/intake/AllInCommand.java

```diff
@@ -30,52 +30,55 @@ public void initialize() {
 	@Override
 	public void execute() {
 		// intake rejecting
-		// if (intakeSubsystem.intakeFrontSeesNote()) {
-		// 	if (!intakeSubsystem.getRejectOverride()) {
-		// 		intakeSubsystem.intakeLeftReject();
-		// 		intakeSubsystem.intakeRightReject();
-		// 	} else {
-		// 		intakeSubsystem.intakeLeftStop();
-		// 		intakeSubsystem.intakeRightStop();
-		// 	}
-
-		// 	if (controls != null && !rumbledIntakeFront) {
-		// 		Commands.race(new RumbleCommand(controls), new WaitCommand(2)).schedule();
-		// 		rumbledIntakeFront = true;
-		// 	}
-		// }
-
-		// // back intake sensor does not exist (but it might?)
-
-		// if (intakeSubsystem.intakeLeftSeesNote()) {
-		// 	if (!intakeSubsystem.getRejectOverride()) {
-		// 		intakeSubsystem.intakeFrontReject();
-		// 		intakeSubsystem.intakeRightReject();
-		// 	} else {
-		// 		intakeSubsystem.intakeFrontStop();
-		// 		intakeSubsystem.intakeRightStop();
-		// 	}
-
-		// 	if (controls != null && !rumbledIntakeLeft) {
-		// 		Commands.race(new RumbleCommand(controls), new WaitCommand(2)).schedule();
-		// 		rumbledIntakeLeft = true;
-		// 	}
-		// }
-
-		// if (intakeSubsystem.intakeRightSeesNote()) {
-		// 	if (!intakeSubsystem.getRejectOverride()) {
-		// 		intakeSubsystem.intakeFrontReject();
-		// 		intakeSubsystem.intakeLeftReject();
-		// 	} else {
-		// 		intakeSubsystem.intakeFrontStop();
-		// 		intakeSubsystem.intakeLeftStop();
-		// 	}
-
-		// 	if (controls != null && !rumbledIntakeRight) {
-		// 		Commands.race(new RumbleCommand(controls), new WaitCommand(2)).schedule();
-		// 		rumbledIntakeRight = true;
-		// 	}
-		// }
+		if (intakeSubsystem.intakeFrontSeesNote()) {
```

Comment:
The intake sensors were taken off, so we should leave this commented out. The index and feeder sensors are still there though.